packages/containers-shared/package.json

```diff
@@ -17,7 +17,6 @@
 	"scripts": {
 		"check:lint": "eslint src --ext ts",
 		"check:type": "tsc -p ./tsconfig.json && pnpm run type:tests",
-		"deploy": "echo 'no deploy'",
```

Comment:
When was this added? It should have broken the `tools/deployments/__tests__/deploy-non-npm-packages.test.ts` test.

Reply:
apparently Emily added it here: https://github.com/cloudflare/workers-sdk/pull/9621/commits/4b228a7eebc4f89b0c374a7e4da860de15902770. I am not sure why tests didn't fail. Maybe same reason as what I was discussing today about ci tests hitting cache?

Reply:
VP PR tests caught this: https://github.com/cloudflare/workers-sdk/actions/runs/15735662603/job/44347571994?pr=9650. But even there I am not 100% sure why the test only fails on Linux. Are we not running this test on all OSs?